frontend/device/gesturedetector.lua

```diff
@@ -50,12 +50,19 @@ local util = require("util")
 local GestureDetector = {
     -- must be initialized with the Input singleton class
     input = nil,
-    -- all the time parameters are in us
+    -- default values (all the time parameters are in us)
     DOUBLE_TAP_INTERVAL = 300 * 1000,
     TWO_FINGER_TAP_DURATION = 300 * 1000,
     HOLD_INTERVAL = 500 * 1000,
     PAN_DELAYED_INTERVAL = 500 * 1000,
     SWIPE_INTERVAL = 900 * 1000,
+    -- all the time parameters are in us
+    ges_double_tap_interval = G_reader_settings:readSetting("ges_double_tap_interval") or 300 * 1000,
```

Comment:
I realize it's duplicated because you can't reference `self.DOUBLE_TAP_INTERVAL` yet but perhaps it'd be better to define them locally above first for clarity? Then it could just be `or DOUBLE_TAP_INTERVAL`.

Reply:
Done.